tools/ci_build/github/linux/docker/Dockerfile_manylinux2014_openvino_multipython

```diff
@@ -0,0 +1,82 @@
+FROM quay.io/pypa/manylinux2014_x86_64:latest
+
+ENV PATH /usr/local/gradle/bin:/opt/rh/devtoolset-10/root/usr/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+ADD scripts /tmp/scripts
+RUN cd /tmp/scripts && /tmp/scripts/manylinux/install_centos.sh && /tmp/scripts/manylinux/install_deps.sh && rm -rf /tmp/scripts
+
+ARG PYTHON_VER_PATH="cp36-cp36m"
+ARG PYTHON_VERSION="3.6"
+ARG BUILD_UID=1001
+ARG BUILD_USER=onnxruntimedev
+ARG OV_DEVICE_PRECISION="CPU_FP32"
+ARG ENABLE_TRAINING=true
+ARG ORT_BRANCH="master"
+RUN adduser --uid $BUILD_UID $BUILD_USER
+WORKDIR /home/$BUILD_USER
+ENV PYTHON_EXE="/opt/python/$PYTHON_VER_PATH/bin/python$PYTHON_VERSION"
+
+RUN yum -y install wget git
+
+# libusb1.0.22
+RUN cd /home/ && wget https://github.com/libusb/libusb/archive/v1.0.22.zip && \
```

Comment:
libusb is GPL. It implies obligations.  I will consult our compliance team on whether we need to register this component to our security&compliance database or not.  @jywu-msft , FYI.

Reply:
This is to enable VPU for OpenVINO Execution Provider..